features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/blocks/api-availability/ajax/api-availability.jag

```diff
@@ -66,10 +66,35 @@ include("/jagg/jagg.jag");
 
 
             dataResult = mod.getDataFromTable(tableName, fullSearchQuery, startindex, count, "api_version", sortOrderBool);
-            countResult = mod.getCountFromTable(tableName, fullSearchQuery);
 
             parsedDataResult = JSON.parse(dataResult);
 
+            var temp1, temp2;
+            var apiDetails, apiProvider, apiName, apiVersion, apiId;
+            var isExists;
+            var modifiedDataResult= new Array();
+
+            var APIManagerFactory = Packages.org.wso2.carbon.apimgt.impl.APIManagerFactory;
+            var APIIdentifier = Packages.org.wso2.carbon.apimgt.api.model.APIIdentifier;
+
+            for (i = 0; i < parsedDataResult.length; i++) {
+                apiDetails= parsedDataResult[i]["values"]["api_version"];
```

Comment:
Can't we directly get API name, version and provider without using splitting logic from parsedDataResult?

Reply:
Data result does not contain separate fields for name, version, provider instead api_version field contains those in provider--apiname:apiversion format. Hence had to use the below splitting logic.